### DISCUSSION TITLE
縱排文字不應臥倒

### DISCUSSION THREAD

**Discussion by heangfat** [linuxdeepin/developer-center#5443] [General | 常规讨论]
工作區左側三个籤「文件瀏覽器」、「符號」、「工程」的文字臥倒了。赶緊立起來吧。又不是拉丁字母，非倒下不可。
正確效果：
![圖片](https://github.com/linuxdeepin/developer-center/assets/20491276/b94c829f-a8fb-4101-a170-c5e0ec509a5b)

**Comment by Decodetalkers** [comment]
@kegechen dtk主题有这种问题。可能之后有时间修？

**Comment by Decodetalkers** [comment]
qt 自己也是这样的。可能可以设置文字方向？我测试了qt自己的平台插件，应该也是一样的，qt应该自己有办法达成这个功能的，这个不是bug

**Comment by Decodetalkers** [comment]
Firefox 和chromium去年才做了，qt没有支持上。你现在可以使用\n\n，或者自绘，在paint 时候旋转90度

https://stackoverflow.com/questions/14299651/qtabwidget-tabs-on-the-vertical-but-text-in-horizontal

比如这个帖子

**Reply by BLumia** [reply]
（我觉得这个是在说那个 ide 项目
